### PR TITLE
Implement reusable transaction card

### PR DESCRIPTION
**Pull request by rdelrosariojr**
## Related Links:
- Issue link

## Definition of Done
- [ ]  Implement a transaction card
    - Icon must be dynamic
    - Text and Background color must be dynamic
    - Value amount color and prefix (`+` or `-`) must be dynamic
- [ ]  Display list based on specific data
    - Expense type
        - pink
        - Icon must be displayed depending on specific category
    - Income type
        - Blue/Purple
        - Icon must be from the currency setting

## Notes:
- I already changed the icon color, exactly similar to the design.

## Screenshots
image
image

## Test View  Points
- [ ] N/A

### CODE REVIEW

File: lib/views/widgets/cards/transaction_card.dart [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+  @override
+  Widget build(BuildContext context) {
+    return Container(
+      child: Row(
+        children: <Widget>[
+          Container(
+            decoration: BoxDecoration(
+              borderRadius: BorderRadius.all(Radius.circular(
+                      8.0) //                 <--- border radius here
+                  ),
```
Comment (AubreyV) [inline lib/views/widgets/cards/transaction_card.dart]: @rdelrosariojr murag this seems off to me.. ang spacing? but if naka format ni acc to flutter then dont mind this na lang. but pls confirm :bow:

File: lib/views/widgets/cards/transaction_card.dart [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+              color: type == 'income' ? Color(0xFFDCE3F9) : Color(0xFFF9DCDE),
```
Comment (AubreyV) [inline lib/views/widgets/cards/transaction_card.dart]: @rdelrosariojr we dont need this colors defined globally?

File: lib/views/widgets/cards/transaction_card.dart (hunk 1 above)
Comment (rdelrosariojr) [reply lib/views/widgets/cards/transaction_card.dart]: @AubreyV haaai :man-bowing:

File: lib/views/widgets/cards/transaction_card.dart (hunk 2 above)
Comment (rdelrosariojr) [reply lib/views/widgets/cards/transaction_card.dart]: @AubreyV I think we can ma'am. I'll add it po.

**Review by AubreyV — APPROVED** [review]
## Approved
Reusable Card for Transaction is same as in design
Both for income and expenses
image
image

**Pull request merged**